src/EnergyPlus/OutsideEnergySources.cc

```diff
@@ -175,7 +175,7 @@ void GetOutsideEnergySourcesInput(EnergyPlusData &state)
             EnergyType = DataPlant::PlantEquipmentType::PurchHotWater;
             heatWaterIndex++;
             thisIndex = heatWaterIndex;
-        } else if (EnergySourceNum > NumDistrictUnitsHeatWater && EnergySourceNum <= NumDistrictUnitsHeatWater + NumDistrictUnitsCool) {
+        } else if (EnergySourceNum <= NumDistrictUnitsHeatWater + NumDistrictUnitsCool) {
```

Comment:
[src/EnergyPlus/OutsideEnergySources.cc:178]:(style),[knownConditionTrueFalse],Condition 'EnergySourceNum>NumDistrictUnitsHeatWater' is always true